runner/dl_checkpoints.sh

```diff
@@ -92,6 +90,8 @@ function download_all_models() {
 
 # Download models only for the live-video-to-video pipeline.
 function download_live_models() {
+  docker pull "${AI_RUNNER_COMFYUI_IMAGE}"
```

Comment:
Pull the runner image only if necessary (`--live` or `--tensorrt`).

Reply:
I'm thinking of rewriting `dl_checkpoints.sh` so instead of running `docker run` the script would be run entirely inside docker